src/hooks/useNYPLBreakpoints.ts

```diff
@@ -42,13 +42,25 @@ const useNYPLBreakpoints = () => {
   ]);
 
   return {
+    // Chakra default
     isLargerThanSmall: layoutSize.isLargerThanSmall,
     isLargerThanMedium: layoutSize.isLargerThanMedium,
-    // NYPL uses the medium 600px breakpoint to determine if the screen is
-    // in the mobile view. This is the recommended boolean value to use.
-    isLargerThanMobile: layoutSize.isLargerThanMedium,
     isLargerThanLarge: layoutSize.isLargerThanLarge,
     isLargerThanXLarge: layoutSize.isLargerThanXLarge,
+
+    // NYPL semantic
+    isLargerThanSmallMobile: layoutSize.isLargerThanSmall,
+    isLargerThanLargeMobile: layoutSize.isLargerThanMedium,
+    isLargerThanSmallTablet: layoutSize.isLargerThanLarge,
+    isLargerThanLargeTablet: layoutSize.isLargerThanXLarge,
+
+    // NYPL uses the medium 768px breakpoint to determine if the screen is in
```

Comment:
This is super useful information. Can this be added in the `Breakpoints.mdx` file for devs to use as the recommended semantic token? I don't mean the `isLargerThanMobile` or `isLargerThanTablet` semantic token names but rather the values. So saying in the doc that it's recommend and most likely that devs will just use  "medium" or "largeMobile".

Reply:
Let me know what you think about this.

Reply:
![image](https://github.com/user-attachments/assets/7feb71e3-75b1-49cf-b3dd-788e1c41e509)